### PR TITLE
version bump. corrected readme for env directory move. unit test and some corrections...

### PR DESCRIPTION
**Pull request by UlysseFG**

### CODE REVIEW

**Review by jleblond — APPROVED** [review]

**Review by jleblond — CHANGES_REQUESTED** [review]
Preferrably this PR could be split into different ones (the readme and env instructions could already pushed), but the unit tests will require more review

File: backend/fms_core/tests/test_models.py [hunk 1]
```diff
@@ -444,3 +446,143 @@ def test_pedigree(self):
...
+        with self.assertRaises(ValidationError):
+            try:
+                ProcessSample.objects.create(process=self.process,
+                                             volume_used=None,
+                                             comment="Test comment")
+            except ValidationError as e:
+                self.assertTrue('source_sample' in e.message_dict)
+                raise e
+
+    def test_missing_volume_used_in_extraction(self):
```
Comment (UlysseFG) [inline backend/fms_core/tests/test_models.py:578]: Nice tests. We could possibly add an identical volume_used test for transfer.

**Review by UlysseFG — COMMENTED** [review]
This looks good ... i suggested another test but this is sugar coating...

**Comment by jleblond** [comment]
Sure, Ill do and merge it to QC afterwards. I will then merge QC into master

**Pull request merged**